macros/edr/dbt_artifacts/dbt_columns/get_dbt_columns_query.sql

```diff
@@ -0,0 +1,66 @@
+{% macro get_dbt_columns_query(is_on_run_end=false) %}
+  {% if is_on_run_end %}
+    {% set get_relation = elementary.get_elementary_relation %}
```

Comment:
Not sure if we care, but just a thought, if a user installs Elementary and doesn't run our models, this would return `None` and then the query would be `from None`.

Reply:
if he does not run our models, this wont run as well